recipes/dragonbox/all/conandata.yml

```
@@ -0,0 +1,9 @@
+sources:
+  "1.1.3":
+    url: "https://github.com/jk-jeon/dragonbox/archive/refs/tags/1.1.3.tar.gz"
+    sha256: "09d63b05e9c594ec423778ab59b7a5aa1d76fdd71d25c7048b0258c4ec9c3384"
+patches:
+  "1.1.3":
+    - patch_file: "patches/1.1.3-fix-include-directory.patch"
+      patch_description: "fix include path"
+      patch_type: "portability"
```

```suggestion
      patch_type: "conan"
```

https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#portability

I don't see a new support, but a change to make it work with Conan.